packages/io/dxf-serializer/index.js

```diff
@@ -78,27 +70,25 @@ EOF
  * @returns {Object} serialized contents, DXF format
  */
 const dxfEntities = (objects, options) => {
-  objects = toArray(objects)
-  let entityContents = objects.map(function (object, i) {
-    if (isCAG(object)) {
-      let paths = object.getOutlinePaths()
+  let entityContents = objects.map((object, i) => {
+    if (geometry.geom2.isA(object)) {
```

Comment:
should this not be filtered like you did in a few other tools ?

Reply:
Added filtering. And error.